spec/classes/init_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 describe 'hosts' do
+  let(:facts) { { :ipaddress => '10.1.2.3',} }
```

Comment:
After configuring spec in my server (thank for your advises about bundler!!), I ran the spec tests and there were 5 tests failed because fqdn_ip_address had a non ip address value. 

I suppose that rspec is unable to infere a value for fqdn_ip_address if both ipaddress and fqdn_ip_address are not set.

That's why I added :ipaddress facts in three different test definitions.